### ISSUE TITLE
What happens if I build this project for Android or iOS?

### ISSUE THREAD

**Issue by shic** [flutter/pinball#479]
I am curious why "I/O Pinball works on Web for desktop and mobile." What happens if I build this project for Android or iOS?

**Comment by atikattar1104** [comment]
Definitely, This Is The Better Idea. Because, I Just Tried Playing The Game And It Is Stuck On Loading Page. I Don't Know How To Raise An Issue On GitHub, But, If You Are Trying To Build This Project For Android, Please Make It Offline (i.e. It Can Run Without Internet Connection).
Thanks
